static/src/stylesheets/module/identity/upsell/_base.scss

```diff
@@ -9,3 +9,31 @@
         font-weight: 100;
     }
 }
+
+.identity-upsell-container {
+    margin-top: $gs-gutter;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.identity-upsell-consent-card {
```

Comment:
we should eventually take this out into its own file but it's not a concern right now!